src/test/resources/features/highlighting/inspections/redundant_coma_inspection.feature

```diff
@@ -0,0 +1,42 @@
+Feature: Inspection for redundant coma in the end of the line
+
+  Scenario Outline: Redundant coma in the end of the line
+    Given a snakemake project
+    Given I open a file "foo.smk" with text
+    """
+    <rule_like> NAME:
+      <section>: "input.txt",
+    """
+    And SmkRedundantComaInspection inspection is enabled
+    Then I expect inspection warning on <,> in <input: "input.txt",> with message
+    """
+    Coma is unnecessary
+    """
+    Examples:
+      | rule_like  | section |
```

Comment:
Please check that inspection also works for `subworkflow` sections

Reply:
Add verification for subworkflow